### PR TITLE
This is my first file :-)

### PR DESCRIPTION
**Pull request by geraldwal**
Don't hesitate to fill in our Venture Lab survey: https://goo.gl/forms/1JY9PTrRkLJdJIPS2 :-)

### CODE REVIEW

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Hi @geraldwal! It seems that your commit is not properly attributed to your name and email, you will probably have to follow https://help.github.com/en/articles/setting-your-username-in-git and then edit your commit using `git commit --amend --set-author="Name <email@student.ie.edu>"` and then `git push --force`, or alternatively open a new pull request.

**Comment by astrojuanlu** [comment]
Hi @geraldwal, did you have any problems changing the authorship of the commit?

**Comment by geraldwal** [comment]
Hi Juan,

thank you for your comment and reminder. So I to change my github username geraldwal to my student e-mail gerald.walravens@student.ie.edu for only your repository?

**Comment by astrojuanlu** [comment]
> for only your repository?

It depends on what you intend to do in the future, but in fact you can set it globally.

**Comment by geraldwal** [comment]
I feel a bit lost, is it because my username says user48 instead of geraldwal or geraldwal instead of gerald.walravens@student.ie.edu ? 

Also the git commit --amend code raises an error I was not able to solve: unknown option `set-author=Name gerald.walravens@student.ie.edu'

**Comment by astrojuanlu** [comment]
> I feel a bit lost, is it because my username says user48 instead of geraldwal or geraldwal instead of gerald.walravens@student.ie.edu ?

The username should say geraldwal, and be linked to https://github.com/geraldwal. You will notice when the avatar matches:

Screenshot_2019-04-07 This is my first file -) by geraldwal · Pull Request #49 · Juanlu001 ie-mbd-advanced-python

> Also the git commit --amend code raises an error I was not able to solve: unknown option `set-author=Name gerald.walravens@student.ie.edu'

I suspect you mistyped the command, can you send a screenshot?

File: GeraldWalravens.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello World, this is geraldwal
```
Comment (astrojuanlu) [inline GeraldWalravens.txt]: Please remove this file :) And keep only the `names/...` one

**Review by astrojuanlu — CHANGES_REQUESTED** [review]

File: GeraldWalravens.txt (hunk 1 above)
Comment (geraldwal) [reply GeraldWalravens.txt]: Should be done. Thanks for your review and feedback!

**Pull request merged**